samples/ra-vpn/redirector-lb/ravpn-ipsec-redirector-config.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: cnfw.cisco.com/v1
+kind: ASAConfiguration
+metadata:
+  name: ravpn-redirector-config
+  namespace: sfcn-system
+  labels:
+    sfcn.cisco.com/service-role: "vpnredirector"
+spec:
+  order: 2
```

Comment:
since you have just one config per serviceRole, each can be set to `1`

Reply:
I copied from existing redirector yaml. Should I make the change there too?

Reply:
ok then don't worry about it. i'll clean it up later